### DISCUSSION TITLE
제가 멘션할 수 있는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by yeo11200** [storycraft/node-kakao#496] [General]
예를 들면 홍보를 하는 사람이 들어왔을때,

제가 부반장이거나 반장이면 강퇴를 바로 시킬순 있지만

그렇지 않을 경우 멘션을 해서 해당 방에 반장과 부반장을 부르고 싶은데 한줄로 멘션을 하고 싶을 때 어떻게 하나요?

sendChat에 array로 보낼 경우 멘션들이 안보내져서 혹시 알려주실수 있을까요?

**Comment by dalgkdbs** [comment]
const chatBuilder = new ChatBuilder().text('짜잔! ');
Array.from(channel.getAllUserInfo())
.filter( info => info.perm == 1 || info.perm == 4)
.map( user => new MentionContent(user))
.forEach( mention => chatBuilder.append(mention));
channel.sendChat(chatBuilder.build(KnownChatType.TEXT));
        
v4로 올라온지 얼마안돼서 일단 예전에 쓰던 방법이랑은 다르게했어요
        
만약 매개변수 arguments 를 오는대로 append를 처리했었다면   new MentionContent 까지 map으로 만들고  append(...Array) 했었을거에요!!

함수에 인자로 배열하나씩 넣고싶다면 ...을 붙여보세여 (이름까먹어서 ㅠㅠ)
